core/Enclave/enclave_hsm.cpp

```diff
@@ -294,43 +327,92 @@ sgx_status_t enclave_create_rsa_key(uint8_t *cmk_blob,
         printf("ecall create_rsa_key cmk_keyblob_size:%lu < key_blob_size:%d.\n", cmk_blob_size, real_keyblob_size);
         return SGX_ERROR_INVALID_PARAMETER;
     }
-        rsa_key_data_t *keyblob = (rsa_key_data_t *)cmk_blob;
+    if (keyspec == NULL || padding_mode == NULL)
+        return SGX_ERROR_INVALID_PARAMETER;
+
+    RSA     *rsa            = NULL;
+    BIGNUM  *bne            = NULL;
```

Comment:
the name is not good!  please following the sgx-ssl rules.

Reply:
copy understood